fms/testing/_internal/model_test_suite.py

```diff
@@ -266,12 +278,26 @@ def _get_signature_params(self) -> Union[int, List[str]]:
         """
         pass
 
+    @property
+    @abc.abstractmethod
+    def _get_signature_optional_params(self) -> Optional[Dict[str, torch.Tensor]]:
```

Comment:
For type hints, now that we're at python 3.10+ everywhere in the stack, we can ditch the Dict And List for dict and list